tests/test_mqttcm_connect.c

```diff
@@ -36,7 +36,7 @@ extern pthread_cond_t mqtt_retry_con;
 extern pthread_mutex_t mqtt_mut;
 extern pthread_cond_t mqtt_con;
 extern rbusHandle_t rbus_handle;
-
+int password_callback(char *buf, int size, int rwflag, void *userdata);
```

Comment:
Add password_callback declaration in ../src/mqttcm_connect.h and avoid defining again here.

Reply:
Same for custom_log_callback also

Reply:
Removed in c file and added in header file